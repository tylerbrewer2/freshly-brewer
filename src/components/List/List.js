import React from 'react';
import styled from 'styled-components';

import SmallHeader from '../SmallHeader';

const ListCore = ({ className, header, children }) => {
  return(
    <section>
      <SmallHeader>{header}</SmallHeader>
      <ul className={className}>
        {children}
      </ul>
    </section>
  )
}

const List = styled(ListCore)`
  list-style: none;
  padding-left: 0;
`

export default List;