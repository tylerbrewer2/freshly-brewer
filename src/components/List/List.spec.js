import React from 'react';
import renderer from 'react-test-renderer';

import List from './List';

it('renders correctly', () => {
  const tree = renderer.create(
    <List header="Test header">
      Test children
    </List>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
