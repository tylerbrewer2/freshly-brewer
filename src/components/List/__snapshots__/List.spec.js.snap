exports[`test renders correctly 1`] = `
<section>
  <h1
    className="sc-bdVaJa hkNTgp">
    Test header
  </h1>
  <ul
    className="sc-bwzfXH bDtYuz">
    Test children
  </ul>
</section>
`;
